doc/sphinx/source/input.rst

```diff
@@ -252,6 +252,8 @@ A list of the datasets for which a CMORizers is available is provided in the fol
 | CERES-SYN1deg                | rlds, rldscs, rlus, rluscs, rlut, rlutcs, rsds, rsdscs, rsus, rsuscs, rsut, rsutcs (3hr)             |   3  | NCL             |
 |                              | rlds, rldscs, rlus, rlut, rlutcs, rsds, rsdt, rsus, rsut, rsutcs (Amon)                              |      |                 |
 +------------------------------+------------------------------------------------------------------------------------------------------+------+-----------------+
+| CFSR                         | clt, hus, hur, prw, rlut, rlutcs, rsut, rsutcs (3hr)                                                 |   2  | Python          |
```

Comment:
```suggestion
| CFSR                         | clt, hus, hur, prw, rlut, rlutcs, rsut, rsutcs (Amon)                                                 |   2  | Python          |
```

Reply:
Thanks Axel for spotting this!
I changed it as suggested.